core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -69,7 +69,7 @@ object NonEmptyLazyList extends NonEmptyLazyListInstances {
     create(ca :+ a)
 
   def apply[A](a: => A, as: A*): NonEmptyLazyList[A] =
-    create(LazyList.concat(LazyList(a), LazyList.from(as)))
+    create(a #:: LazyList.from(as))
```

Comment:
`LazyList(a)` calls a varargs factory method, creating an `Array` wrapping `a`, and thus destroying its laziness. instead, we use `#::` to preserve the laziness of `a`

Reply:
it's worth noting that, unless a `LazyList` is explicitly passed as `as` (rather than varargs), this method is not particularly lazy, as `as` is not lazy (specifically, `as` is a wrapped `Array` if varargs are used).

Reply:
actually, this is no worse than `LazyList.apply`. nvm